### ISSUE TITLE
Can't find Change Log for @datocms/cma-client-browser

### ISSUE THREAD

**Issue by arsinclair** [datocms/js-rest-api-clients#9]
Where can I find a changelog for _@datocms/cma-client-browser_?

**Comment by marcelofinamorvieira** [comment]
Hey @arsinclair We have a product wide change-log right here https://www.datocms.com/product-updates, but not one specifically for the @datocms/cma-client-browser, however we haven't deprecated or changed any calls from that package since its release, are you having any issue with a specific request?

**Comment by arsinclair** [comment]
Hey, I'm trying to upgrade to a newer version of the client and I always review the change logs. Is there no plan to publish one specifically for the _@datocms/cma-client-browser_?

**Comment by marcelofinamorvieira** [comment]
At the moment there isn't a specific one unfortunately :(
But I do agree with you that it would be nice to start the publication of one for each package.
I'll check to see if we can start doing so in the next releases :)

**Comment by arsinclair** [comment]
Thank you.

**Comment by kkor** [comment]
Same for @datocms/cma-client-node! It got bumped a major version without a change log? 😬

**Comment by Jokinen** [comment]
I have to agree. Upgrading these packages always takes time, because you end up searching for a changelog, but it doesn't exist. Have worked with Dato for a few years and I've ran into this issue a few times. For instance, currently, I need to update `@datocms/cma-client-node` to resolve a security vulnerability. The newest version is one major higher and I'd like to understand what the breaking change was, but can't find something that would allow me to get a grasp of what's changing.

It's a strong convention in JS land to bump the major on breaking changes only. That's why I have to assume there is one if there has not been communication to suggest something else. As a stopgap, could it be worth it to asses whether the documentation for these packages can be amended to describe what the version system is (if understanding it is helpful for package users).

**Comment by stefanoverna** [comment]
@Jokinen https://github.com/datocms/js-rest-api-clients/releases/tag/v3.0.0? Is this too little I guess? The breaking change is that now we directly use the `fetch()` provided by node, without relying on third-party libraries.

**Comment by Jokinen** [comment]
@stefanoverna That would work really well. But AFAIK the current version is 5? And the latest release notes are for version 3. So it would still be unclear to me what the breaking changes are for 4 and 5.
